tests/antares/services/api_services/test_study_api.py

```diff
@@ -330,3 +336,60 @@ def test_create_link_same_area(self):
             match=f"Could not create the link {area} / {area}: A link cannot start and end at the same area",
         ):
             self.study.create_link(area_from=area, area_to=area)
+
+    def test_run_and_wait_antares_simulation(self):
+        parameters = AntaresSimulationParameters(solver=Solver.COIN, nb_cpu=2, unzip_output=True, presolve=False)
+        with requests_mock.Mocker() as mocker:
+            run_url = f"https://antares.com/api/v1/launcher/run/{self.study_id}"
+            job_id = "1234-g6z17"
+            mocker.post(run_url, json={"id": job_id}, status_code=200)
+
+            job_url = f"https://antares.com/api/v1/launcher/jobs/{job_id}"
+            response_list = [
```

Comment:
I'm not sure to understand this. The output shouldn't be a list when calling this endpoint. We can discuss it later.